### PR TITLE
fix(spaces):

### PR DESCRIPTION
**Pull request by daniel7764**
-Refactored all the spaces and placing of fields across the forms

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx [hunk 1]
```diff
@@ -124,128 +123,139 @@ const ClinicalDetails: React.FC = (): JSX.Element => {
...
+                            if (selectedCityName === '') {
+                                updateIsolationAddressOnCityChange('');
+                                setIsolationStreetName('');
+                            }
+                        }}
+                        renderInput={(params) =>
+                            <TextField
+                                {...params}
+                                placeholder='עיר'
+                                className={classes.textField}
```
Comment (sandymash) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx:161]: property textField doesnt exist on new styles

**Review by sandymash — APPROVED** [review]

**Review by OrelKaravani — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx [hunk 2]
```diff
@@ -90,176 +90,189 @@ const ClinicalDetails: React.FC = (): JSX.Element => {
...
-                <Grid item xs={12}>
                 </Grid>
-                <Grid item xs={2}>
+            </Collapse>
+            <Grid spacing={3} container className={classes.containerGrid} justify='flex-start' alignItems='center'>
+                <Grid item xs={2} className={classes.fieldLabel}>
                     <Typography>
                         <b>
-                            האם בעייתי לקיים בידוד:
+                            כתובת לאשפוז ביתי:
```
Comment (sandymash) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx:144]: image
addres fields overlap in ckinical details tab

File: client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/FlightsForm/FlightsForm.tsx [hunk 3]
```diff
@@ -1,125 +1,134 @@
...
-          }
-          placeholder="הזן מספר טיסה"
-        />
-      </FormRowWithInput>
-    </Grid>
-  );
+    return (
+        <Grid className={formClasses.form} container justify="flex-start">
+            <Grid item xs={2}>
+                <FormRowWithInput fieldName="יעד:">
```
Comment (sandymash) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/FlightsForm/FlightsForm.tsx:26]: image
fields are too small here

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/ContactForm/ContactForm.tsx [hunk 4]
```diff
@@ -81,73 +81,77 @@ const ContactForm: React.FC<Props> = (props: Props): JSX.Element => {
...
                 </Grid>
-                <Grid item xs={4}>
+                <Grid item xs={4} className={classes.contactAdditionalDetails}>
                     <FormInput fieldName={contactTypeField}>
                         <FormControl fullWidth>
                             <InputLabel>סוג מגע</InputLabel>
                             <Select
                                 test-id={'contactType'}
+                                className={classes.newContactField}
                                 label='סוג מגע'
```
Comment (sandymash) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/ContactForm/ContactForm.tsx:128]: image
label is in weird position

**Pull request closed without merging**